README.md

```diff
@@ -69,7 +92,9 @@ Don't see an example listed? [Try Pulumi AI](https://www.pulumi.com/ai) and use
 
 ### TypeScript
 
-[(Click here to go back to the main list)](#outline)
+[(Click here to go back to the library)](#example-library)
```

Comment:
It's a link, so no need to say "click here":

```suggestion
[🔝 Back to the list](#example-library)
```

I'd also suggest putting these below the tables rather than above them.